CHANGELOG.md

```diff
@@ -126,6 +126,15 @@
 
   ([PR #1330](https://github.com/alphagov/govuk-frontend/pull/1330))
 
+- Removes `govuk-grid-row` mixin
+
+  https://github.com/alphagov/govuk-frontend/pull/1090 copied govuk-grid-row
+  mixin to create a new concrete `.govuk-grid-row` class and marked the mixin as deprecated.
+
+  **ACTION TO BE TAKEN**: Remove any references to `govuk-grid-row` mixin and use `.govuk-grid-row` class.
```

Comment:
This seems a little shouty considering I'd be very surprised if anyone's actually using it. Can we consider using the same format we used for the colour palette change?

> To migrate you'll need to remove any references to `govuk-grid-row` mixin and use `.govuk-grid-row` class instead.